datacontract.schema.json

```diff
@@ -272,21 +272,51 @@
                 "type": "string",
                 "description": "An optional string describing the server."
```

Comment:
I’ve corrected the other descriptions but for this one, I was unsure whether it referred to the AWS Account Id or name. 

In my examples I used the Id, can you confirm if that’s correct?